### ISSUE TITLE
agent not started on demand on fedora

### ISSUE THREAD

**Issue by akostadinov** [elementary/pantheon-agent-geoclue2#30]
I read #26. But agent is not started automatically for me. If I run agent manually and start `redshift` while it runs, then it works. But if agent exits, then redshift fails to work. Any explanation what should be starting the agent automatically?

I'm running Fedora 28, XFCE, pantheon-agent-geoclue2-1.0.1-1.fc28.x86_64

**Comment by tintou** [comment]
D-Bus should be starting the Agent on-demand

**Comment by akostadinov** [comment]
And what should tell D-Bus what to start? I only see `/etc/dbus-1/system.d/org.freedesktop.GeoClue2.Agent.conf` with this content:
```
<busconfig>
  <policy user="geoclue">
    <allow send_interface="org.freedesktop.GeoClue2.Agent"
           send_path="/org/freedesktop/GeoClue2/Agent"/>
    <allow send_interface="org.freedesktop.DBus.Properties"
           send_path="/org/freedesktop/GeoClue2/Agent"/>
  </policy>

  <policy user="root">
    <allow send_interface="org.freedesktop.GeoClue2.Agent"
           send_path="/org/freedesktop/GeoClue2/Agent"/>
    <allow send_interface="org.freedesktop.DBus.Properties"
           send_path="/org/freedesktop/GeoClue2/Agent"/>
  </policy>
</busconfig>
```

Or should agent register to D-Bus? Excuse my ignorance but I don't have much clue about D-Bus.

**Comment by decathorpe** [comment]
This should be fixed with version 1.0.2, which is on its way to the fedora repos.
